msg/ServoReport.msg

```diff
@@ -0,0 +1,9 @@
+uint64 timestamp					# time since system start (microseconds)
+
+float32 servo_position					# feedback position of servo. [-1;1]
+float32 servo_speed					# feedback speed of servo [deg/s]
+float32 servo_acceleration				# feedback of acceleration
+uint8 load_pct						# Load percentage
```

Comment:
Is this equal to Torque?

Reply:
I dont think so, because if I look at the graphs, during constant speed,  load_pct is constant and my expectation would be a first order behavior for the torque.

It could be a constant current/torque for a given speed command, but not too sure.